app/controllers/application_controller.rb

```diff
@@ -24,7 +24,7 @@ def error(status_code, exception = nil)
     if exception && defined? GovukError
       GovukError.notify exception
     end
-    render status: status_code, text: "#{status_code} error"
+    head(status_code)
```

Comment:
Although this is valid is it now inconsistent with most of our other apps that return some text with the header? I think the issue with the previous version of the code is the use of text keyword argument rather than plain.

Reply:
Hmm, I'll take another look. Thanks!

Reply:
Cool, changing `text:` for `plain:` worked 👍